dashboard/app/models/survey_result.rb

```diff
@@ -40,10 +40,9 @@ class SurveyResult < ApplicationRecord
     diversity_black
     diversity_hispanic
     diversity_asian
-    diversity_asian
-    diversity_native
+    diversity_hawaiian
     diversity_american_indian
-    diversity_other
+    diversity_tr
```

Comment:
Is there any reason for serialized attributes to still include keys used in older responses? I don't think we read from this model anywhere, so it probably doesn't matter.

Reply:
You know what?  I don't think this constant is used anywhere.  I'm just going to delete the whole list and see what happens.  Thanks!

Reply:
Interesting, I was thinking serialized_attrs was a Rails validation thing that determined what serialized attributes could be saved. Sounds like you already figured this out but apparently it's just [homebaked](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/serialized_properties.rb#L50) and gives you a list of serialized properties, like `SurveyResult.serialized_attrs`? Also [here's some unrelated helpful variable naming](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/serialized_properties.rb#L42). 😆

Reply:
Heh, I often see `klass` used as a variable name when referring to a class, not sure if I've seen `clazz` before.  (You can't call the variable `class` because it's a reserved keyword.)

I took another look and realized that I misunderstood how the survey values were getting into the database.  I had thought that the JSON was just being sanitized and then saved to the db, but it's actually being parsed and then each value is being stored as a serialized attribute on the model.  Thanks for pointing this out!  (Side note: This design couples the db model to the survey questions which seems odd to me.)

To answer your original question, I'll leave the old values in the list with a note that they're not used anymore in case anyone wants to access the old values via the model.